Added link to the banner of Hacktoberfest

thomasbnt opened the pull request:
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Just adding a link to https://www.hacksquad.dev on the banner which is in the README to have a better context.

scopsy commented:
@iampearceman I actually think that you want to send it to the new language page right? If so could you provide the link to @thomasbnt so he can add the appropriate link?

eelliot908 commented:
You Dey jones

eelliot908 reviewed (commented):
I'm not aware

thomasbnt commented on README.md:
```diff
@@ -1,12 +1,12 @@
   
-![hacktoberfest-readme-dark](https://user-images.githubusercontent.com/63902456/191170980-c010daeb-f8ff-454d-bda1-543695b32c80.jpeg#gh-dark-mode-only)
+[![hacktoberfest-readme-dark](https://user-images.githubusercontent.com/63902456/191170980-c010daeb-f8ff-454d-bda1-543695b32c80.jpeg#gh-dark-mode-only)](https://www.hacksquad.dev)
 
 </div>
 
 <div align="center">
   
- ![hacktoberfest-readme-light](https://user-images.githubusercontent.com/63902456/191171065-d8f349e5-7250-43ff-8078-fa02dd859bc8.jpeg#gh-light-mode-only)
+ [![hacktoberfest-readme-light](https://user-images.githubusercontent.com/63902456/191171065-d8f349e5-7250-43ff-8078-fa02dd859bc8.jpeg#gh-light-mode-only)](https://www.hacksquad.dev)
```
Hmm I just seen the theme wasn't applied with the markdown has a link. 
https://github.com/thomasbnt/novu/blob/added-link-to-banner-hacktoberfest-thomasbnt/README.md

p-fernandez reviewed (approved):
🌟 
@iampearceman if @thomasbnt has already implemented the links you provided we are good to go here

scopsy commented:
@thomasbnt could you please resolve conflict from main? we updated the banner image URL with a different text. And also, could you make the link go to: https://novu.co/hacktoberfest/ instead of hacksquad?

thomasbnt commented:
> @thomasbnt could you please resolve conflict from main? we updated the banner image URL with a different text. And also, could you make the link go to: [novu.co/hacktoberfest](https://novu.co/hacktoberfest/) instead of hacksquad?

Hello ! Changed to https://novu.co/hacktoberfest/ and replaced images URL 👌🏼

scopsy reviewed (approved):
Thank you so much for your contribution! Really appreciate it 🙏
